clients/curl/example.sh

```diff
@@ -40,3 +40,6 @@ curl -v -u $AUTH http://$BSHOST/camli/enumerate-blobs?limit=1
 
 # List offset -- 200 with list of no blobs
 curl -v -u $AUTH http://$BSHOST/camli/enumerate-blobs?after=sha1-126249fd8c18cbb5312a5705746a2af87fba9538
+
+# Search for all images
+echo '{"expression":"is:image","describe":{"rules":[{"attrs":["camliContent"]}]}}' | curl -v  --data-binary  @- -u $AUTH http://localhost:3179/my-search/camli/search/query
```

Comment:
Since the other examples use `$BSHOST`; maybe we should use something like `$SEARCHHOST`

Reply:
Also, just in case you didnt know, you could use the perkeep cli for this too `pk search "is:image"`

Reply:
> Also, just in case you didnt know, you could use the perkeep cli for this too `pk search "is:image"`

Yep and I think that is useful too!

I'm just trying to record and share how to do it over http.  I couldn't find anything about search here https://perkeep.org/doc/protocol/.   I'm not prepared to fill that gap at this point, but I would like to move towards that.  FWIW, issue #1037 is asking for that as well.

Reply:
Yeah i think its a useful change, what do you think about using `$SEARCHHOST` similar to `$BSHOST` in the other examples?

Reply:
Added